source/performance.txt

```diff
@@ -83,27 +90,27 @@ The :guilabel:`Performance` tab displays various stats:
        operations correspond to a given moment in the charts; i.e. as
        you move over the charts, the corresponding slowest operations
        are displayed in the table. To view the details of a slow
-       operation or :ref:`kill the operation <kill-long-ops>`, click on
+       operation or :ref:`stop the operation <kill-long-ops>`, click on
        the operation.
 
 .. _kill-long-ops:
 
-Kill Slow Operations
+Stop Slow Operations
```

Comment:
In this section I replaced "Kill" with "Stop" to align with our style guide recommendations. https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#k